api/graphql/schema.graphql

```diff
@@ -280,7 +280,9 @@ type User {
   "All albums owned by this user"
   albums: [Album!]! @isAdmin
   "Top level albums owned by this user"
-  rootAlbums: [Album!]! @isAdmin
+  rootAlbums(
+    order: Ordering,
```

Comment:
Do you plan to change the order in UI later? I don't think currently web UI has a way to change the order of the path. I think we could update the SQL query only, but not adding an order parameter in API.

Reply:
I didn't plan any other updates to this. I changed the pr, removed the changes from the API.